packages/extension-chakra-storefront/test-paths.txt

```diff
@@ -0,0 +1,9 @@
+# to keep the CI clean, we have initial skip all test in the `npm run test` command
```

Comment:
For a clean CI, Temporary keep this file until we have all the tests renabled to avoid manually skip all the tests we haven't fixed to.

Reply:
Have you thought about using the .skip method on the tests. You can simply do a folder wide find and replace. The benefits here are:

1. We get notified about skipped tests so we are actively reminded that we have to re-enable them. The current solution does kinda of the opposite of and we are maintaining a list of items to test and jest doesn't know anything about skipping.
2. No additional overhead of keeping track of these test-path file and update to the script definition.

Reply:
@bendvc We have thoughts of tests, this means we have to manually edit hundred of tests files to skip the tests?